sound/soc/codecs/hdac_hdmi.c

```diff
@@ -1331,6 +1333,7 @@ static int hdac_hdmi_add_pin(struct hdac_device *hdev, hda_nid_t nid)
 
 #define INTEL_VENDOR_NID 0x08
 #define INTEL_GLK_VENDOR_NID 0x0b
+#define INTEL_ICL_VENDOR_NID 0x02
```

Comment:
NID is vendor node is very strange based on my old knowledge. Where do you find the information? I need to have a check.

Reply:
@libinyang I will send you the link of the spec.

Reply:
I think we need to revisit this. This 0x2 NID is correct but not limited to ICL, so we should have the NID information defined in less-ICL specific terms, e.g. INTEL_VENDOR_NID_0x2 - same for GLK.

Reply:
@plbossart How about using magic number directly? The define is only used in the _dra_data struct and we can know it is a vendor nid from the field's name (.vendor_nid).

Reply:
we want to keep the info that this is an INTEL VENDOR thingy. This is lost if you use 0x2, people might assume it's a standard value instead of a vendor one. So yes while the define brings no value from a compilation perspective, it's semantically useful for the code reviews.